ci-operator/step-registry/netobserv/perf-test/metrics-upload/netobserv-perf-test-metrics-upload-ref.yaml

```diff
@@ -17,6 +17,12 @@ ref:
   - name: INSTALLATION_SOURCE
     default: "Source"
     documentation: "Source that netobserv was installed from: Source or Internal"
+  - name: MULTISTAGE_PARAM_OVERRIDE_SAMPLING
+    default: ""
+    documentation: Set to override sampling rate
```

Comment:
this isn't needed in this step, right?

Reply:
I have exported the vars in the test-deploy script, it can be read by the metrics upload script right?

Reply:
that's a good question, I am not sure if these are available across the steps. I dug into this a bit, it looks like 
`PROW_JOB_ID` env var is available across all steps. instead of checking for each MULTISTAGE* var, may be we can query `https://prow.ci.openshift.org/prowjob?prowjob=${PROW_JOB_ID})` and parse its response using `yq`, for e.g.:
```
curl "https://prow.ci.openshift.org/prowjob?prowjob=e4420ea1-3e85-4ba3-89b6-e43994c5a725" 2> /dev/null | yq '.metadata.annotations["ci.openshift.io/executor"]'
gangway
```
if it returns `gangway`, we don't upload as baseline. Only thing is we'd have to add `yq` to our netobserv-perfscale Docker image, it's currently not there.

Reply:
Implemented based of https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh#L439